tests/manage/z_cluster/cluster_expansion/test_node_expansion.py

```diff
@@ -25,6 +25,8 @@ def test_add_node(self):
         """
         dt = config.ENV_DATA['deployment_type']
         if dt == 'ipi':
```

Comment:
Any plan in future to use this TC for other platform node add too ? if so will be good to have the params parametrized

Reply:
on the way ,in my next pr